samples/rtl/rtl_tasklet_scalar.py

```diff
@@ -44,8 +44,10 @@
         end else if (valid_i) begin // case: load a 
             b <= a;
             ready_o <= 1'b0;
-        end else // case: increment counter b
-            b <= b + 1; 
+        end if (b < 100) // case: increment counter b
```

Comment:
@andreaskuster Did you mean `end else if`?

Reply:
Yes, good catch! Thanks.
These typos should hopefully be catched by the unit tests soon :)